src/pull_request.ts

```diff
@@ -104,17 +104,22 @@ export async function handlePullRequest() {
     prDescription: pull_request.body || "",
     prSummary: summary.description,
   });
+  console.log("Review: ", review.review);
+  console.log("Comments: ", review.comments);
   info(`reviewed pull request`);
```

Comment:
Consider removing or commenting out the console logs before merging to keep the production code clean. If logging is necessary for debugging, consider using a logging library instead.